packages/builder/src/App.svelte

```diff
@@ -1,13 +1,16 @@
 <script>
   import { Router } from "@roxi/routify"
   import { routes } from "../.routify/routes"
-  import { NotificationDisplay } from "@budibase/bbui"
+  import { NotificationDisplay, BannerDisplay } from "@budibase/bbui"
   import { parse, stringify } from "qs"
   import HelpIcon from "components/common/HelpIcon.svelte"
 
   const queryHandler = { parse, stringify }
 </script>
 
+<div class="banner-container" />
+<BannerDisplay />
```

Comment:
This seems a bit strange to me - rendering a DOM node and then immediately after rendering a svelte portal which targets that DOM node. Why don't we just rename `BannerDisplay` to `StatusBanner`, remove the portal altogether and the `.banner-container` div, and just have the `StatusBanner` render a normal banner with status info in it's current location? That would achieve the same result but be a bit less complicated, and more explicit that it's intended to be used for status messages rather than a generic banner.

Reply:
Sounds like a good suggestion @aptkingston . I had a go at this but hit a roadblock around pointer events, the pointer isn't able to interact with with banner without the portal. Attached is the diff of what I tried, if you have any other suggestions of how to work around this I'd be up for trying them out. Gonna merge this one for now so that we can release the two fixes in the release